austation/code/modules/mining/minebot.dm

```diff
@@ -0,0 +1,74 @@
+#define MINEDRONE_COLLECT 1
+#define MINEDRONE_ATTACK 2
+
+/mob/living/simple_animal/hostile/mining_drone/Initialize()
+	. = ..()
+	weather_immunities += "ash" // no damage from ash storms
+	stored_gun.overheat_time = 10
+
+
+/obj/item/slimepotion/slime/sentience/mining
+	name = "minebot AI upgrade"
+	desc = "Can be used to grant sentience to minebots."
+	icon_state = "door_electronics"
+	icon = 'icons/obj/module.dmi'
+	sentience_type = SENTIENCE_MINEBOT
+
+/obj/item/slimepotion/slime/sentience/mining/after_success(mob/living/user, mob/living/simple_animal/SM)
+	return // player drones are able to be upgraded with better HP and attack, unlike on Bee.  Removes all the code that nerfed player minebots
```

Comment:
```suggestion
	return // player drones are able to be upgraded with better HP and attack. Allows drones to be upgraded
```

Reply:
slightly less snark, plus I'm pretty sure they weren't nerfed, they were always this way

Reply:
Bee does nerf them once a player takes control.  Minebots can only be upgraded if the stat is still at its base number. Bee gives the minebot a very tiny buff to health and melee so that upgrades can't be used on them and outright nerfs the pka by making it fire a second slower, whether it was improved or not.

I'll think on a way to get this across with less snark.